packages/@ama-sdk/core/cli/update-spec-from-npm.cts

```diff
@@ -58,8 +61,11 @@ void (async () => {
 
   let specDestinationPath = output;
   if (!specDestinationPath) {
-    const specSourceExtension = extname(specSourcePath);
-    specDestinationPath = `./openapi${specSourceExtension}`;
+    let specSourceExtension = extname(specSourcePath);
+    if (specSourceExtension === `.${SPEC_YAML_EXTENSION}` || specSourceExtension === `.${SPEC_YML_EXTENSION}`) {
```

Comment:
do you really need to specify the second condition since the variable will remain unchanged?
```suggestion
    if (specSourceExtension === `.${SPEC_YAML_EXTENSION}`) {
```

Reply:
true :)